pkg/policy/validate.go

```diff
@@ -905,10 +904,13 @@ func ruleOnlyDealsWithResourceMetaData(rule kyverno.Rule) bool {
 	return true
 }
 
-func validateResources(rule kyverno.Rule) (string, error) {
+func validateResources(path *field.Path, rule kyverno.Rule) (string, error) {
 	// validate userInfo in match and exclude
-	if path, err := validateUserInfo(rule); err != nil {
-		return fmt.Sprintf("resources.%s", path), err
+	if errs := rule.MatchResources.UserInfo.Validate(path.Child("match")); len(errs) != 0 {
```

Comment:
We also need to validate `rule.MatchResources.Any/All[].UserInfo` which was introduced after the original match/exclude blocks.

Reply:
Yeah, I remember a discussion about that in Slack. Is it ok if I add those checks when migrating `MatchResources` and `ExcludeResources` logic ?